src/gui/addnewtorrentdialog.h

```diff
@@ -112,6 +116,7 @@ private slots:
     QByteArray m_headerState;
     int m_oldIndex;
     QScopedPointer<TorrentFileGuard> m_torrentGuard;
+    BitTorrent::AddTorrentParams torrentParams;
```

Comment:
If there is a member, it has to be initialised in the constructor, but not in `show()` function.